src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs

```diff
@@ -279,6 +332,31 @@ private static CalcResultSummaryProducerDisposalFees GetProducerRow(
             };
         }
 
+        //section bad debt total
```

Comment:
Please move all new functions to a new folder and call it TotalBillPartA

Reply:
@udaydefra - I think doing this will create more work for now, as the newly written functions are dependent on input from other functions.
Happy to discuss.

Reply:
Thanks John i has same opinion.